docs/source/stubs.rst

```diff
@@ -25,7 +25,9 @@ Here is an overview of how to create a stub file:
     $ export MYPYPATH=~/work/myproject/stubs
 
 Use the normal Python file name conventions for modules, e.g. ``csv.pyi``
-for module ``csv``. Use a subdirectory with ``__init__.pyi`` for packages.
+for module ``csv``. Use a subdirectory with ``__init__.pyi`` for packages. Note
+that PEP 561 stub-only packages must be installed, and may not be pointed at
```

Comment:
Maybe link to the PEP 561 page here? I think this is the first time we mention it on this page.

Reply:
Sure, that is a good idea.

Reply:
(This is not done yet.)